kasa/smart/modules/alarm.py

```diff
@@ -136,10 +162,51 @@ def source(self) -> str | None:
         src = self._device.sys_info["in_alarm_source"]
         return src if src else None
 
-    async def play(self) -> dict:
-        """Play alarm."""
-        return await self.call("play_alarm")
+    async def play(
+        self,
+        *,
+        duration: int | None = None,
+        volume: Literal["low", "normal", "high"] | None = None,
```

Comment:
Noting here that the `smartcam` has values 1-10 for volume. Somehow we want to think about how this will work with a common module. My view is that we probably want to move towards `int` for volume as it's more generic, that would be breaking, but maybe this PR is the right time to do it. Or perhaps the volume could be:
```py
volume: str | int | None = None,
```
and just rely on the ValueErrors.

Reply:
Having this as a 1-3 range sounds good to me. Requires a breaking change in homeassistant, too.

Reply:
Maybe keep `str | int` which won't break HA and then after we move HA to common alarm interface we can drop the str?